components/OutcropDetailsPage/index.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { capitalizeFirstLetter } from "../../helper/functions";
+
+import {
+  Container,
+  Form,
+  Input,
+  TextArea,
+  Button,
+  List,
+  ListItem,
+  Headline,
+  Header,
+} from "./OutcropDetailsPage.styled";
+
+const OutcropDetailsPage = ({
+  formState,
+  onFormSubmit,
+  onInputChange,
+  submittedData,
+  title,
+}) => {
+  const dataKeys = [
+    "gesteinsart",
+    "gesteinsklasse",
+    "schichtung",
+    "faltung",
+    "mineralien",
+    "allgemeines",
+    "interpretation",
+  ];
+
+  return (
+    <>
+      <Header>
+        <Headline>Aufschluss 1 Titel</Headline>
+      </Header>
+      <Container>
+        <Form onSubmit={onFormSubmit}>
+          {dataKeys.map((key) => (
+            <div
+              className={`form-field${
```

Comment:
try not to use classnames, since we are going with styled components. 
check [your repo here](https://github.com/neuefische/muc-web-23-3/blob/main/sessions/react-styled-components/react-styled-components.md#adapting-based-on-props) for inspiration